plugin/path_checkout_sets.go

```diff
@@ -114,41 +128,36 @@ func (b *backend) operationReserveCreate(ctx context.Context, req *logical.Reque
 		return logical.ErrorResponse(`"service_account_names" must be provided`), nil
 	}
 	// Ensure these service accounts aren't already managed by another check-out set.
-	var alreadyManagedErr error
+	var userErrs error
+	var resultingServiceAccountNames []string
 	for _, serviceAccountName := range serviceAccountNames {
-		if _, err := b.checkOutHandler.Status(ctx, req.Storage, serviceAccountName); err != nil {
-			if err == ErrNotFound {
-				// This is what we want to see.
+		if isUserErr, err := b.checkInNewServiceAccount(ctx, req.Storage, serviceAccountName); err != nil {
+			if isUserErr {
```

Comment:
I'm not sure we'd want a partially completely operation to succeed. Is there a reason to use that here? As a counter example, if when creating a role one provided an invalid parameter, we wouldn't typically create the role using the the valid parameter, but would rather error out the whole request.

Reply:
Aren't errors being batched here prior to committing?  I think its helpful to know all errors instead of working through each one to get the next error, since a set can have `n` accounts.

Reply:
This no longer exists due to no longer doing partial updates. Marking as resolved.